.github/workflows/doc_build.yml

```diff
@@ -9,14 +9,14 @@ on:
 jobs:
   lint:
     name: html + linkcheck build
-    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
-
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
 
     - name: Set up latest Python
       uses: actions/setup-python@v4
+      with:
+        python-version: "*"
```

Comment:
Does * mean latest available?

Reply:
I don't find `*` documented at https://github.com/actions/setup-python. They have:

```yml
python-version: '3.x' # Version range or exact version of a Python version to use, using SemVer's version range syntax
```

For the Black PR, I saw one of the [other workflows](https://github.com/psf/black/blob/6c1bd08f16b636de38b92aeb2e0a1e8ebef0a0b1/.github/workflows/upload_binary.yml#L39) already had `*` so I copied and pasted from there!

That was added in https://github.com/psf/black/pull/2260 in commit [psf/black@`2fcc183` (#2260)](https://github.com/psf/black/pull/2260/commits/2fcc183a07e703c8d6166a7bf533a463a885beb9), and there's no docs reference. Maybe @felix-hilden remembers where it came from? Lucky guess?

Reply:
Stuff works so we can merge and make better if we learn :D Thanks!

Reply:
The [doc](https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python#using-a-specific-python-version) is linked in the PR! But the gist is: it's a SemVer range, so indeed the latest. `3.x` would not allow Python 4 versions, which obviously is a *huge* issue 😉